lib/main.dart

```diff
@@ -24,13 +24,28 @@ import "package:flutter/material.dart";
 import 'package:flutter_displaymode/flutter_displaymode.dart';
 import 'package:logging/logging.dart';
 import 'package:path_provider/path_provider.dart';
+import 'package:privacy_screen/privacy_screen.dart';
 
 final _logger = Logger("main");
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   final savedThemeMode = await AdaptiveTheme.getThemeMode();
   await _runInForeground(savedThemeMode);
+  await PrivacyScreen.instance.enable(
+    iosOptions: const PrivacyIosOptions(
+      enablePrivacy: true,
+      privacyImageName: "LaunchImage",
+      autoLockAfterSeconds: 5,
```

Comment:
Do we need `autoLock` since we already have a separate lockscreen?

Reply:
Also, how does this affect the behavior of the app when the existing lock screen is turned on?

Reply:
will remove the autolock. Its irrelevant